Portfolio/index.css

```diff
@@ -1 +1,88 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+  
+    margin:  0px;

```

Comment:
The value for padding should include a unit such as 'px', 'em', etc. For example, padding: 10px;